From 992723e3712a09037338aa9e5506a080e24d1642 Mon Sep 17 00:00:00 2001
From: Stijn Tintel <stijn@linux-ipv6.be>
Date: Mon, 25 Sep 2017 08:09:01 +0300
Subject: [PATCH] ipset: fix build with musl

Include sys/types.h for u_int8_t and define _GNU_SOURCE for musl to
expose it.

Fixes: 54802b2c2826 ("Report if the option is supported by a newer kernel release")
Signed-off-by: Stijn Tintel <stijn@linux-ipv6.be>
Signed-off-by: Jozsef Kadlecsik <kadlec@blackhole.kfki.hu>
---
 src/ipset.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ipset.c b/src/ipset.c
index 79f56b8..8d70abc 100644
--- a/src/ipset.c
+++ b/src/ipset.c
@@ -14,6 +14,8 @@
 #include <stdio.h>			/* fprintf, fgets */
 #include <stdlib.h>			/* exit */
 #include <string.h>			/* str* */
+#define _GNU_SOURCE
+#include <sys/types.h>			/* u_int8_t */
 
 #include <config.h>
 
-- 
2.13.5

